packages/react-native/React/CoreModules/RCTDeviceInfo.mm

```diff
@@ -111,14 +111,14 @@ static BOOL RCTIsIPhoneNotched()
 #if !TARGET_OS_OSX // [macOS]
 NSDictionary *RCTExportedDimensions(CGFloat fontScale)
 #else // [macOS
-static NSDictionary *RCTExportedDimensions(RCTPlatformView *rootView)
```

Comment:
I removed the `rootView` argument since we were never passing this in and we always lookup current window.

Reply:
Was there no way to pass one in without using the bridge? And in 0.71 and earlier, did we actually pass one in?

Reply:
iOS always passed in a bridge, but macOS was passing `nil` for `rootView` since the beginning it seems.

https://github.com/microsoft/react-native-macos/blame/ce04f7ed7a599e9ed3ba986d938fdcaad6289514/React/CoreModules/RCTDeviceInfo.mm#L142-L145

Reply:
If iOS is passing in a bridge, can we do that as well and get the root view from there? Presumably, the iOS code will work / get updated to work with bridge less, so whatever bridge reference is getting passed might be a proxy or get refactored away over time.

Reply:
I removed the forked implementation for `macOS` since we can use the `iOS` one and pass in our own `fontScale`

Reply:
We talked offline, but I'll summarize here:
It would be ideal if this method actually did take a root view. However, I'm not sure we have ever actually passed it in. Furthermore, this method (used by the `useWindowDimensions` hook is ultimately now multi-window or multi-root-view friendly, and likely will never be. So, we might as well be consistent with iOS, if this method is already one I'd not recommend using.